integration_testing/features/admin/admin-find-user.feature

```diff
@@ -16,7 +16,7 @@ Feature: Admin find users
     When I click the left arrow (previous page) button
     Then I am back on the first page of the list of facility users
     When I keep clicking the right arrow to find a user
-    Then I see the numbers of pages correctly increasing in increments of 10
+    Then I see the numbers of pages correctly increasing in increments of 10 # or 30, if th enumber of users is close to 1000
```

Comment:
typo error `th enumber`

Reply:
Not important, will correct in future updates!